### PR TITLE
added package: sysbench

### PR DESCRIPTION
**Pull request by SeanDougherty**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Added support for the benchmarking package 'sysbench'. 
In support of task #37101243 on ADO

###### Change Log  

- Addition of sysbench

###### Does this affect the toolchain?  

NO

###### Test Methodology

Local build

### CODE REVIEW

File: SPECS/sysbench/sysbench.spec [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+URL:            https://github.com/akopytov/sysbench/
+Source0:        https://github.com/akopytov/%{name}/archive/%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  automake
+BuildRequires:  libaio-devel
+BuildRequires:  libtool
+BuildRequires:  make
+BuildRequires:  mariadb-devel
+BuildRequires:  pkgconfig
+BuildRequires:  postgresql-devel
+BuildRequires:  python
```
Comment (oliviacrain) [inline SPECS/sysbench/sysbench.spec:16]: Can we change this build requirement to specifically reference `python3`?

File: SPECS/sysbench/sysbench.spec [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+- cpu: a simple CPU benchmark
+- memory: a memory access benchmark
+- threads: a thread-based scheduler benchmark
+- mutex: a POSIX mutex benchmark
+
+%prep
+%setup -q
+
+%build
+export CFLAGS="%{optflags}"
```
Comment (oliviacrain) [inline SPECS/sysbench/sysbench.spec:39]: Nit: Exporting `CFLAGS` shouldn't be necessary, as the definition is automatically inserted as part of the `%build` template.

```suggestion
```

File: SPECS/sysbench/sysbench.spec [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+           --without-gcc-arch
+
+%make_build
+
+%install
+%make_install
+rm -f %{buildroot}%{_docdir}/sysbench/manual.html
+
+%check
+make test
```
Comment (oliviacrain) [inline SPECS/sysbench/sysbench.spec:53]: Nit: Could be the following to enable simultaneous jobs and go a little faster:

```suggestion
%make_build test
```

File: SPECS/sysbench/sysbench.spec (hunk 1 above)
Comment (SeanDougherty) [reply SPECS/sysbench/sysbench.spec]: Tested, and change has been applied via git commit --amend!

File: SPECS/sysbench/sysbench.spec (hunk 3 above)
Comment (SeanDougherty) [reply SPECS/sysbench/sysbench.spec]: Tested, and change has been applied via git commit --amend!

**Review by oliviacrain — APPROVED** [review]
Looks great!

**Review by christopherco — APPROVED** [review]

**Pull request merged**